extensions/interactions/TextInput/TextInputValidationService.js

```diff
@@ -17,12 +17,36 @@
  */
 
 oppia.factory('TextInputValidationService', [
-  'baseInteractionValidationService',
-  function(baseInteractionValidationService) {
+  'WARNING_TYPES', 'baseInteractionValidationService',
+  function(WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+        var warningsList = [];
+        baseInteractionValidationService.requireCustomizationArguments(
+          customizationArgs,
+          ['placeholder', 'rows']);
+
+        var rows = customizationArgs.rows.value;
```

Comment:
Maybe use angular.isNumber() and something from here: https://stackoverflow.com/questions/3885817/how-do-i-check-that-a-number-is-float-or-integer